app/api/settings/route.ts

```
@@ -11,18 +11,42 @@ export async function PUT(req: NextRequest) {
     }
     const userId = session.user.id;
     const body = await req.json();
-    const { enableEmailCapture } = body;
+    const { enableEmailCapture, webhookUrl } = body;
 
-    if (typeof enableEmailCapture !== "boolean") {
-      return NextResponse.json({ error: "enableEmailCapture must be a boolean" }, { status: 400 });
+    let updateData: any = {};
```

ESLint

[error] 'updateData' is never reassigned. Use 'const' instead.

(prefer-const)